app/controllers/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -78,6 +78,15 @@ def reassign_hostgroup
         end
       end
 
+      api :PUT, "/hosts/bulk/disassociate", N_("Disassociate compute resources")
+      param_group :bulk_host_ids
+      def disassociate
+        BulkHostsManager.new(hosts: @hosts).disassociate
+        process_response(true, { :message => n_("Updated %{count} host: Disassociated from VM",
+          "Updated %{count} hosts: Disassociated from VM",
```

Comment:
Should we keep this consistent with the action description?
```suggestion
        process_response(true, { :message => n_("Updated %{count} host: Disassociated from compute resource",
          "Updated %{count} hosts: Disassociated from compute resource",
```